stistools/ocrreject.py

```diff
@@ -323,15 +318,15 @@ def ocrreject(input, output,
                 arglist.append("-t")
             arglist.extend(optional_args)
 
```

Comment:
@stscirij -

Found a logic bug due to the indentation here.  It should now process multiple files when `all=False`.

(Moved comment up to clarify)